spec/unit_spec_helper.rb

```diff
@@ -14,6 +14,8 @@
     config.infer_spec_type_from_file_location!
   end
 
+  config.example_status_persistence_file_path = "spec/examples.txt"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.